archetypes/README.md

```diff
@@ -1,19 +1,23 @@
-# The Infinispan Archetypes project
+# Infinispan Archetypes
```

Comment:
Now that we have proper documentation, we can probably remove this file entirely. I included it it in the initial PR mainly as a primer for yourself.

Reply:
maybe it's good to have something there. I've included a link to the getting started guide. perhaps that is all that's needed.